widgetastic_manageiq/__init__.py

```diff
@@ -2941,7 +2941,10 @@ def _current_page_elements(self):
         else:
             entities = self._invoke_cmd('get_all_items')
             for entity in entities:
-                elements.append({'name': entity['item']['cells']['Name'],
+                # Floating Ip view has an issue. it doesn't have Name though it should
+                name = entity['item']['cells'].get('Name') or entity['item']['cells'].get(
```

Comment:
**optional** can we factor out the fetching of the cells, so the expression for the name would use the single dict directly?

**required** we should use an `KeyError` raising expressionfor the deferred part, in order to ensure name has a value

Reply:
I woldn't want to do that. JS entities come with several dicts with properties. Some of properties have same name. I encountered several issues because of name collision when I combined all those dictionaries into one dict.